hms-app/src/TestIdeaData.js

```diff
@@ -0,0 +1,18 @@
+export default [
+    {
+        titel:"idea 1",
+        description:"really a good idea ",
+        favNumber: 2
+    },
+    {
+        titel:"idea 2",
+        description:"just a better idea ",
+        favNumber: 8
+    },
+    {
+        titel:"idea 3",
+        description:"absolute the best idea in the world ",
+        favNumber: 5
+    },
+
+]
```

Comment:
it's a weird old fashioned thing but when possible files should always end in a newline

Reply:
thanks for the hint